docs/process/testing.md

```diff
@@ -1,17 +1,17 @@
 ### Download Desktop
   - [ ] User can download latest (Mac & Windows) Desktop from https://desktop.github.com/    
     - [ ] Mac: https://central.github.com/deployments/desktop/desktop/latest/darwin
-      - [ ] Homebrew package manager: $ brew cask install github-desktop
+      - [ ] Homebrew package manager: `$ brew cask install github-desktop`
```

Comment:
If you remove the preceding `$`, people can copy and paste more easily into their terminals.

Reply:
k